bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcPanel.scala

```diff
@@ -152,11 +152,11 @@ class ItcSpectroscopyPanel(val owner: EdIteratorFolder) extends ItcPanel {
     case SPComponentType.INSTRUMENT_FLAMINGOS2  => true
     case SPComponentType.INSTRUMENT_GMOS        => true
     case SPComponentType.INSTRUMENT_GMOSSOUTH   => true
-    case SPComponentType.INSTRUMENT_GNIRS       => true
-    case SPComponentType.INSTRUMENT_MICHELLE    => true
-    case SPComponentType.INSTRUMENT_NIFS        => true
+    case SPComponentType.INSTRUMENT_GNIRS       => false // will add support at later stage
+    case SPComponentType.INSTRUMENT_MICHELLE    => false // may or may not be supported in OT at some point
+    case SPComponentType.INSTRUMENT_NIFS        => false // will add support at later stage
     case SPComponentType.INSTRUMENT_NIRI        => true
-    case SPComponentType.INSTRUMENT_TRECS       => true
+    case SPComponentType.INSTRUMENT_TRECS       => false // may or may not be supported in OT at some point
     case _                                      => false
   }
 }
```

Comment:
Targets may be copied between observations using different instruments, so probably best to retain and show the Source information in all cases rather than have it lost or hidden some of the time.